sdks/csharp/eventRoute_operations.cs

```diff
@@ -1,38 +1,51 @@
-// ------------------ CREATE EVENT ROUTE (Basic) ---------------------
-// <CreateEventRoute>
-string eventFilter = "$eventType = 'DigitalTwinTelemetryMessages' or $eventType = 'DigitalTwinLifecycleNotification'";
-var er = new DigitalTwinsEventRoute("endpointName", eventFilter);
-await client.CreateOrReplaceEventRouteAsync("routeId", er);
-// </CreateEventRoute>
+using Azure;
+using Azure.DigitalTwins.Core;
+using System;
+using System.Threading.Tasks;
 
-// ------------------ CREATE, LIST, AND DELETE EVENT ROUTE (Full sample) ---------------------
-// <FullEventRouteSample>
-private static async Task CreateEventRouteAsync(DigitalTwinsClient client, string routeId, DigitalTwinsEventRoute er)
+namespace DigitalTwins_Samples
 {
-    try
+    public class EventRouteOperationsSample
     {
-        Console.WriteLine("Create a route: testRoute1");
-
-        // Make a filter that passes everything
-        er.Filter = "true";
-        await client.CreateOrReplaceEventRouteAsync(routeId, er);
-        Console.WriteLine("Create route succeeded.");
-
-        // List routes
-        AsyncPageable<DigitalTwinsEventRoute> results = client.GetEventRoutesAsync();
-        await foreach (DigitalTwinsEventRoute route in results)
+        public static async Task CreateEventRouteBasic(DigitalTwinsClient client)
         {
-            Console.WriteLine($"Route {route.Id} to endpoint {route.EndpointName} with filter {route.Filter} ");
+            // ------------------ CREATE EVENT ROUTE (Basic) ---------------------
+            // <CreateEventRoute>
+            string eventFilter = "$eventType = 'DigitalTwinTelemetryMessages' or $eventType = 'DigitalTwinLifecycleNotification'";
+            var er = new DigitalTwinsEventRoute("endpointName", eventFilter);
+            await client.CreateOrReplaceEventRouteAsync("routeId", er);
+            // </CreateEventRoute>
         }
 
-        // Delete route created earlier
-        Console.WriteLine($"Deleting route {routeId}:");
-        await client.DeleteEventRouteAsync(routeId);
+        // ------------------ CREATE, LIST, AND DELETE EVENT ROUTE (Full sample) ---------------------
+        // <FullEventRouteSample>
+        private static async Task CreateEventRouteAsync(DigitalTwinsClient client, string routeId, DigitalTwinsEventRoute er)
```

Comment:
@cippero  Should this method be public as well per the comment on line 8?

Reply:
Yes, I'll update it with the next commit.